v1/feed.js

```diff
@@ -120,7 +120,9 @@ Feed.prototype.aggregated = function(hyper, req) {
             };
             // populate its body
             Object.keys(result).forEach(function(key) {
-                finalResult.body[key] = result[key].body;
+                if (!!result[key].body && Object.keys(result[key].body).length) {
```

Comment:
Minor: The `!!` seems to be unnecessary here, and might be less clear than a plain "truthy" test.